peer-signer/README.md

```diff
@@ -0,0 +1,47 @@
+# Peer Signer
+This tool is intended to verify a potential candidate for testnet validator.
+Package can be split into two parts, `signer` and `verifier`.
+
+
+Signer part will be used by the candidate.
+With this program candidate is going to sign a secret message provided by us to prove they own the p2p secret they claim to own.
+
+
+Verifier part will be used by Aleph Zero team.
+With this program we are going to check whether supplied public key derives to supplied peer id, signature of secret message is correct and this peer is connected and up to date with our network.
+
+## Step one: signature generation by Validator candidate
+To generate signature, candidate needs a generated p2p address.
```

Comment:
how do I generate such address? when I pass `p2p_secret` file that was generated from `./scripts/run_nodes.sh` I get:
```
thread 'main' panicked at 'Incorrect secret format. Failed to create a secret key.: DecodingError { msg: "Ed25519 secret key", source: Some(signature::Error { source: Some(SecretKey must be 32 bytes in length) }) }', src/signer.rs:30:10
```

Reply:
This is caused by our `./scripts/run_nodes.sh` generating p2p secret in hex encoded format. Currently script for running external validator that we are developing saves it in raw format. I changed signer, to be generic in sense that it first tries to generate Secret Key  from raw bytes and if it fails, signer tries to first decode from hex and after that generate Secret Key.